src/published-components.js

```diff
@@ -4,6 +4,10 @@ const fs = require("fs");
 const publishedComponents = {
   // Don't remove next line
   // plop_marker:published-components
+  MenuTitle: "/src/components/Menu/MenuTitle.jsx",
+  Divider: "/src/components/Divider.jsx",
```

Comment:
This is my fault (the plop had an issue)
should be "/src/components/Divider/Divider.jsx"